### PR TITLE
Do not check editor != reviewer if to review not enabled (fixes #131)

### LINKED ISSUE #131: If review is not enabled, it should not check editor != reviewer
Otherwise it is a strong breaking change when just upgrading the lib without enabling workflows

### PR DESCRIPTION
**Pull request by leplatrem**
Fixes #131

@tarekziade  | @Natim  r?

### CODE REVIEW

File: kinto_signer/listeners.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def check_collection_status(event, resources, group_check_enabled,
             if requires_review and to_review_enabled:
                 raise_invalid(message="Collection not reviewed")
 
-            if old_collection.get(FIELD_LAST_EDITOR) == current_user_id:
+            if to_review_enabled and old_collection.get(FIELD_LAST_EDITOR) == current_user_id:  # NOQA
```
Comment (Natim) [inline kinto_signer/listeners.py]: Why do you need the ``# NOQA`` tag here? Shouldn't we add a test to validate that?

Comment (leplatrem) [inline kinto_signer/listeners.py]: `# NOQA` if for line-length (not coverage)

I added a test that covers the case, in one test suite the spec is *test_editor_cannot_be_reviewer* and in another the spec is *test_editor_can_be_reviewer*

Comment (Natim) [inline kinto_signer/listeners.py]: Then we should allow greater line length in the tox.ini file.

Comment (leplatrem) [inline kinto_signer/listeners.py]: If there's only one, maybe not. I'll split the line then.

**Pull request merged**